odoo/addons/base/models/res_bank.py

```diff
@@ -90,6 +91,14 @@ def _get_supported_account_types(self):
         'The combination Account Number/Partner must be unique.'
     )]
 
+    @api.model
+    def _get_view(self, view_id=None, view_type='form', **options):
+        arch, view = super()._get_view(view_id, view_type, **options)
+        if clearing_number_label := self.env.company.country_id.clearing_number_label:
```

Comment:
should we see `ABA/Routing` label when we are in an US company but checking EU res.bank records?